libs/novui/src/components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,59 @@
+import { Tabs as ExternalTabs, type TabsProps as ExternalTabsProps } from '@mantine/core';
+import { forwardRef, ReactNode } from 'react';
+import { CoreProps, LocalizedMessage } from 'src/types';
+import { css, cx } from '../../../styled-system/css';
```

Comment:
I suggest adding a path alias to the novui tsconfig for `@/styled-system` access in a separate PR.